flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -148,7 +149,10 @@
 	/** Service to contend for and retrieve the leadership of JM and RM */
 	private final HighAvailabilityServices highAvailabilityServices;
 
-	/** Blob cache manager used across jobs */
+	/** Blob server used across jobs */
+	private final BlobServer blobServer;
```

Comment:
Actually, I'd rather let the services involved in the life cycle of the BLOBs known what they are dealing with than adding empty `register/releaseJob` methods to the `BlobService`/`BlobServer`. Once we move the `BlobServer` out of the `JobMaster`, we can  change the class of the parameters and also remove the `register/releaseJob` calls.